packages/components/src/panel/README.md

```diff
@@ -127,6 +127,20 @@ An icon to be shown next to the `PanelBody` title.
 -   Type: `String`
 -   Required: No
 
+###### iconOpen
+
+An icon to be shown as the toggle button within the `PanelBody` title when open.
+
+-   Type: `String`
+-   Required: No
```

Comment:
These props are not `String`s — let's update the description as follow:

```suggestion
An icon to be shown as the toggle button within the `PanelBody` title, in its expanded state. See [the `@wordpress/icons` package](https://github.com/WordPress/gutenberg/blob/trunk/packages/icons/README.md) for a list of available icons.

-   Type: `React.ReactNode`
-   Required: No
-   Default: `chevronUp` icon from `@wordpress/icons`
```

Reply:
@ciampo - Sorry, wasn't sure what type they should be, I copied from the lines above documenting the existing `icon` prop which also said `string`.

Should I update the original `icon` prop documentation to match?

Reply:
> Should I update the original icon prop documentation to match?

Yes please. I believe the `icon` prop being a `String` is a leftover from earlier times when `dashicons` were the main way to provide icons.